### PR TITLE
[ZH] Update `Gameplay/Accuracy`

### PR DESCRIPTION
**Pull request by Andy-K-Sparklight**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

This PR updates outdated translation `Gameplay/Accuracy`.

### CODE REVIEW

File: wiki/Gameplay/Accuracy/zh.md [hunk 1]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
-在 osu!catch 中，准确度以收集的非转盘物件数量总和除以非转盘物件数量总和计算。所有物件的价值相同，除了香蕉，因为它是转盘的一部分。
+在 osu!catch 模式中，准确度以收集的非转盘物件数量的总和，除以非转盘物件数量总和计算。所有物件的价值相同，但香蕉除外，因为它算是一种转盘。
+
+*[API](https://github.com/ppy/osu-wiki/blob/master/wiki/osu!api) 用户请注意：*
 
-*对 API 用户的提醒：若要自行计算osu!catch的准确度，注意droplets的数量记为 `count50` ，而错过的droplets数量记为 `countkatu` *
+- `count100` 代表接住的水果串（Drops）数目。
+- `count50` 代表接住的水果串节点（Droplets）数目。
+- `countMiss` 代表漏掉的水果（Fruits）数与水果串（Drops）数的*总和*。
+- `countKatu` 代表漏掉的水果串节点（Droplets）数目。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:39]:
```suggestion
- `count100` 代表接住的中果数。
- `count50` 代表接住的小果数。
- `countMiss` 代表漏掉的大果与中果数的*总和*。
- `countKatu` 代表漏掉的小果数。
```
ref: https://osu.ppy.sh/wiki/zh/Gameplay/Hit_object/Juice_stream

File: wiki/Gameplay/Accuracy/zh.md [hunk 2]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
 
 ![准确度 = 0.5(GOOD的数量 + GREAT的数量) / (BAD的数量 + GOOD的数量 + GREAT的数量)](img/accuracy_taiko.png "osu!taiko的准确度公式")
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:24]:
```suggestion
![准确度 = (良的数量 + 0.5 \* 可的数量 ) / (良的数量 + 可的数量 + 不可的数量)](img/accuracy_taiko.png "osu!taiko的准确度公式")
```

目前，良、可、不可已在 cn osu!taiko 中通用，应翻译。
且这个地方有漏译。

File: wiki/Gameplay/Accuracy/zh.md [hunk 3]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
 
-在 osu!taiko 中，准确度以物件准确度（击打物件时的时间差）除以已计分的物件总量计算。物件准确度被归类为 GREAT (良) (记为100%)，GOOD (可) (记为50%，也就是一半)，以及 MISS/BAD (不可) (记为0%, 且会打断combo)。Drum rolls 和 spinners 不影响准确度。
+在 osu!taiko 模式中，准确度以物件准确度（击打物件时的时间差）除以已计分的物件总量计算。物件准确度被归类为 GREAT（良，记为 100%），GOOD（可，记为 50%），以及 MISS/BAD（不可，记为 0%, 且连击会断掉)。长条（Drum rolls）和转盘（Spinners）不会影响准确度。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:26]:
```suggestion
在 osu!taiko 模式中，准确度以物件准确度（击打物件时的时间偏差）除以已计分的物件总量计算。物件准确度被归类为良（GREAT，记为 100%），可（GOOD，记为 50%），以及不可（MISS/BAD，记为 0%，且会断掉连击)。长条（Drum rolls）和转盘（Spinners）不会影响准确度。
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 4]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
 ![准确度 = (droplets的数量 + drops的数量 + 水果的数量) / (错过的droplets的数量 + 错过的drops的数量 + 错过的水果的数量 + droplets的数量 + drops的数量 + 水果的数量)](img/accuracy_catch.png "osu!catch的准确度公式")
 
-在 osu!catch 中，准确度以收集的非转盘物件数量总和除以非转盘物件数量总和计算。所有物件的价值相同，除了香蕉，因为它是转盘的一部分。
+在 osu!catch 模式中，准确度以收集的非转盘物件数量的总和，除以非转盘物件数量总和计算。所有物件的价值相同，但香蕉除外，因为它算是一种转盘。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:32]:
```suggestion
在 osu!catch 模式中，准确度以收集的非转盘物件数量的总和，除以非转盘物件数量总和计算。所有物件的价值相同，但香蕉除外，因为它属于转盘的一部分。
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 5]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
-![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 200 \* 200的数量 + 300 \* 300的数量 + 300 \* MAX的数量) / 300(0的数量 + 50的数量 + 100的数量 + 200的数量 + 300的数量 + MAX的数量)](img/accuracy_mania.png "osu!mania的准确度公式")
+在 osu!mania 模式中, 准确度计算方式类似 [osu!](#osu!) 模式。但是，彩 300 的权重大小会受到 ScoreV2 模组的影响。
+
+若没有开启 ScoreV2，彩 300 和金 300 的权重一样，都是 300：
+
+![Accuracy = (300 \* (number of MAXs + number of 300s) + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (300 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania with ScoreV1")
+
+若开启了 ScoreV2，彩 300 的权重会增加到 305：
+
+![Accuracy = 305 \* number of MAXs + 300 \* number of 300s + 200 \* number of 200s + 100 \* number of 100s + 50 \* number of 50s) / (305 \* (number of MAXs + number of 300s + number of 200s + number of 100s + number of 50s + number of misses))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania with ScoreV2")
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:52]:
```suggestion
![Accuracy = (300 \* (彩 300 数量 + 300 数量) + 200 \* 200 数量 + 100 \* 100 数量 + 50 \* 50 数量) / (300 \* (彩 300 数量 + 300 数量 + 200 数量 + 100 数量 + 50 数量 + 失误数量))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania with ScoreV1")

若开启了 ScoreV2，彩 300 的权重会增加到 305：

![Accuracy = 305 \* 彩 300 数量 + 300 \* 300 数量 + 200 \* 200 数量 + 100 \* 100 数量 + 50 \* 50 数量) / (305 \* (彩 300 数量 + 300 数量 + 200 数量 + 100 数量 + 50 数量 + 失误数量))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania with ScoreV2")
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 6]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
+- `countMiss` 代表漏掉的水果（Fruits）数与水果串（Drops）数的*总和*。
+- `countKatu` 代表漏掉的水果串节点（Droplets）数目。
+- `countGeki` 的值不能用于计算准确率。它只代表连击末尾的水果数。
 
 ### ![](/wiki/shared/mode/mania.png) osu!mania
 
-![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 200 \* 200的数量 + 300 \* 300的数量 + 300 \* MAX的数量) / 300(0的数量 + 50的数量 + 100的数量 + 200的数量 + 300的数量 + MAX的数量)](img/accuracy_mania.png "osu!mania的准确度公式")
+在 osu!mania 模式中, 准确度计算方式类似 [osu!](#osu!) 模式。但是，彩 300 的权重大小会受到 ScoreV2 模组的影响。
+
+若没有开启 ScoreV2，彩 300 和金 300 的权重一样，都是 300：
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:46]:
```suggestion
若没有开启 ScoreV2，彩 300 和普通 300 的权重一样，都是 300：
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 7]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
+#### 偏差（Error）
 
-`Error` 将永远显示两个值，代表提前击打和延迟击打时间的平均值。谱面的 [Overall Difficulty](/wiki/Beatmapping/Overall_difficulty) 值越高，游玩谱面时得到好的分数所需要的 Error 值也越低。
+`Error` 将永远显示两个值，代表提前击打和延迟击打时间的平均值。谱面的[总体难度](/wiki/Beatmapping/Overall_difficulty)越高，想得到好的分数的偏差就得越小。
 
-#### Unstable rate
+#### 不稳定度（Unstable rate）
 
-`Unstable Rate` 代表击打计时的连续性，越低的值越好 （顶尖玩家往往保持在100以下）。 注意该值衡量的是连续性，因此连续地提前15毫秒击打将和连续准时击打相同。 公式基本上是击打误差 （以毫秒为单位） 乘以10。 [示例代码](https://gist.github.com/peppy/3a11cb58c856b6af7c1916422f668899) 可以作为参考，展示了 osu-stable 如何计算 Unstable Rate 值。
+`Unstable Rate` 代表打击时间的不稳定程度，越低越好（顶尖玩家往往保持在 100 以下）。该值衡量的是打击时间的稳定性，因此总是提前 15 毫秒击打，和总是准时击打相比，其不稳定度是一样的。按公式算出来的结果基本上就是击打误差（单位：毫秒）乘 10。[这段代码](https://gist.github.com/peppy/3a11cb58c856b6af7c1916422f668899)展示了 osu! Stable 版是如何计算不稳定度的，可以作为参考。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:79]:
```suggestion
`不稳定度` 简称为 UR，即打击时间的不稳定程度，越低越好（顶尖玩家往往保持在 100 以下）。该值衡量的是打击时间的稳定性，因此总是提前 15 毫秒击打，和总是准时击打相比，其不稳定度是一样的。按公式算出来的结果基本上就是击打误差（单位：毫秒）乘 10。[这段代码](https://gist.github.com/peppy/3a11cb58c856b6af7c1916422f668899)展示了 osu! 稳定版是如何计算不稳定度的，可以作为参考。
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 8]
```diff
@@ -29,50 +23,67 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
-*注意：附加信息仅可在游玩谱面或者观看回放后查看。退出 [结算界面](/wiki/Client/Interface#排名) 后，这些信息不会被保存。*
+*注意：附加信息仅可在游玩谱面或者观看回放后查看。退出[结算界面](/wiki/Client/Interface#排名)后，这些信息不会被保存。*
 
 ### 准确度
 
-光标悬停在表现图上方时， 将显示一个包含 `Error` 和 `Unstable Rate` 评级的提示框
+光标悬停在表现图上方时， 将显示一个包含 `偏差（Error）` 和 `不稳定度（Unstable Rate）` 评估的提示框。
 
-由于 [DT](/wiki/Game_modifier/Double_Time) (Double Time) 和 [HT](/wiki/Game_modifier/Half_Time) (Half Time) 的实现方法， Error 和 Unstable Rate 的值将会被乘以谱面的速度变化率。 当使用 DT 时， 要获得真正的 Unstable Rate，将结果除以 1.5 即可。 以此类推，当使用 HT mod 游玩时将结果乘以 1.33 即可。
+由于 [DT](/wiki/Game_modifier/Double_Time) (Double Time) 和 [HT](/wiki/Game_modifier/Half_Time) (Half Time) 模组的实现方法，偏差和不稳定度的值将会被乘以谱面的速度变化率。当使用 DT 模组时，要获得不稳定度的合理值，将结果除以 1.5 即可。类似的，当使用 HT 模组时将结果乘 1.33 即可。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:71]:
```suggestion
由于 [DT](/wiki/Game_modifier/Double_Time) (Double Time) 和 [HT](/wiki/Game_modifier/Half_Time) (Half Time) 模组的实现方法，偏差和不稳定度的值将会乘上谱面的速度变化率。当使用 DT 模组时，要获得合理的不稳定度，将结果除以 1.5 即可。类似的，当使用 HT 模组时将结果乘 1.33 即可。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
need fix.
需要修改这些地方。

File: wiki/Gameplay/Accuracy/zh.md [hunk 9]
```diff
@@ -27,52 +21,69 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
-![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 200 \* 200的数量 + 300 \* 300的数量 + 300 \* MAX的数量) / 300(0的数量 + 50的数量 + 100的数量 + 200的数量 + 300的数量 + MAX的数量)](img/accuracy_mania.png "osu!mania的准确度公式")
+在 osu!mania 模式中, 准确度计算方式类似 [osu!](#osu!) 模式。但是，彩 300 的权重大小会受到 ScoreV2 模组的影响。
+
+若没有开启 ScoreV2，彩 300 和普通 300 的权重一样，都是 300：
+
+![Accuracy = (300 \* (彩 300 数量 + 300 数量) + 200 \* 200 数量 + 100 \* 100 数量 + 50 \* 50 数量) / (300 \* (彩 300 数量 + 300 数量 + 200 数量 + 100 数量 + 50 数量 + 失误数量))](img/accuracy_mania_updated_score_v1.png "Accuracy formula for osu!mania with ScoreV1")
+
+若开启了 ScoreV2，彩 300 的权重会增加到 305：
+
+![Accuracy = 305 \* 彩 300 数量 + 300 \* 300 数量 + 200 \* 200 数量 + 100 \* 100 数量 + 50 \* 50 数量) / (305 \* (彩 300 数量 + 300 数量 + 200 数量 + 100 数量 + 50 数量 + 失误数量))](img/accuracy_mania_updated_score_v2.png "Accuracy formula for osu!mania with ScoreV2")
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:52]: 改一下后面的翻译

File: wiki/Gameplay/Accuracy/zh.md [hunk 10]
```diff
@@ -27,52 +21,69 @@ outdated_since: a4d801f4dbb4b86901f63773b7993647eb5b7ba8
...
 
-![准确度 = 0.5(GOOD的数量 + GREAT的数量) / (BAD的数量 + GOOD的数量 + GREAT的数量)](img/accuracy_taiko.png "osu!taiko的准确度公式")
+![准确度 = (良的数量 + 0.5 \* 可的数量 ) / (良的数量 + 可的数量 + 不可的数量)](img/accuracy_taiko.png "osu!taiko的准确度公式")
 
-在 osu!taiko 中，准确度以物件准确度（击打物件时的时间差）除以已计分的物件总量计算。物件准确度被归类为 GREAT (良) (记为100%)，GOOD (可) (记为50%，也就是一半)，以及 MISS/BAD (不可) (记为0%, 且会打断combo)。Drum rolls 和 spinners 不影响准确度。
+在 osu!taiko 模式中，准确度以物件准确度（击打物件时的时间偏差）除以已计分的物件总量计算。物件准确度被归类为良（GREAT，记为 100%），可（GOOD，记为 50%），以及不可（MISS/BAD，记为 0%，且会断掉连击)。长条（Drum rolls）和转盘（Spinners）不会影响准确度。
 
 ### ![](/wiki/shared/mode/catch.png) osu!catch
 
 ![准确度 = (droplets的数量 + drops的数量 + 水果的数量) / (错过的droplets的数量 + 错过的drops的数量 + 错过的水果的数量 + droplets的数量 + drops的数量 + 水果的数量)](img/accuracy_catch.png "osu!catch的准确度公式")
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:30]:
```suggestion
![准确度 = (droplets的数量 + drops的数量 + 水果的数量) / (错过的droplets的数量 + 错过的drops的数量 + 错过的水果的数量 + droplets的数量 + drops的数量 + 水果的数量)](img/accuracy_catch.png "osu!catch 的准确度公式")
```

File: wiki/Gameplay/Accuracy/zh.md [hunk 11]
```diff
@@ -1,22 +1,16 @@
...
+准确率（也称准确度）用于刻画玩家打击[物件](/wiki/Hit_object)的时间与标准时间有多接近，亦即打得多准。玩家有三种准确度：谱面的准确度，取决于谱面的物件得分；总准确度，它是为让更好的得分更加突出而权衡的；[表现分（pp）](/wiki/Performance_points)准确度，取决于提交的得分的准确度。
 
-准确度是对玩家准时击中[物件](/wiki/Hit_object)的能力的百分数评估。玩家有三种准确度：谱面的准确度，这取决于所得的命中分数；总准确度，它是为让更好的得分更加突出而权衡的；[Performance Points (pp)](/wiki/Performance_points)准确度，它取决于提交的得分的准确度。
-
-## 游戏模式
+## 各模式下的准确率计算方法
 
 ### ![](/wiki/shared/mode/osu.png) osu!
 
 ![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 300 \* 300的数量) / 300( 0的数量 + 50的数量 + 100的数量 + 300的数量)](img/accuracy_osu.png "osu!的准确度公式")
```
Comment (YumeMuzi) [inline wiki/Gameplay/Accuracy/zh.md:9]:
```suggestion
![准确度 = (50 \* 50 的数量 + 100 \* 100 的数量 + 300 \* 300的数量) / 300 * (0 的数量 + 50 的数量 + 100 的数量 + 300 的数量)](img/accuracy_osu.png "osu! 的准确度公式")
```

**Comment by Andy-K-Sparklight** [comment]
The missing part has been updated. Thanks for pointing out. Please check if there are any mistakes.
缺失的翻译已经补全，感谢指出。敬请复检是否存在其它的错误。

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**